app/views/patients/edit.html.erb

```diff
@@ -5,13 +5,11 @@
 </div>
 
 <div id="patient_detail" class="row">
-  <div>
-    <div id="menu" class="col-sm-3">
-      <%= render 'patients/menu', patient: @patient %>
-    </div>
+  <div id="menu" class="col-3">
+    <%= render 'patients/menu', patient: @patient %>
   </div>
 
-  <div id="sections" class="col-sm-9 tab-content">
+  <div id="sections" class="col tab-content">
```

Comment:
interesting name for an id name, should be a class?

Reply:
probably true! I'll leave a todo